backEnd/routes/play.js

```diff
@@ -1,13 +1,40 @@
 const express = require("express");
 const mySqlConnect = require("./connection");
-const Joi = require('joi');
+const Joi = require("joi");
 const router = express.Router();
-const Axios = require ('axios');
+const Axios = require("axios");
+var childProcess = require("child_process");
+
+//I Did not test locally.
+function runScript(scriptPath, jsonArg, errLogCallback) {
+  var invoked = false;
+
+  var process = childProcess.fork(scriptPath, {
+    execArgv: ["--" + jsonArg]
```

Comment:
I think you may need to do "--json " + jsonArg as the command to run it normally is:

node Playback.js --json <json str>